.github/workflows/image-arm-pr.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
   docker:
     uses: ./.github/workflows/reusable-docker-arm-build.yaml
     with:
-      flavor: ubuntu
+      flavor: opensuse-leap-arm-rpi
```

Comment:
I changed the flavor here for some weird reason on the alpine PR..... this brings it back